circuit_puzzles/atom_announcer.clsp

```diff
@@ -3,12 +3,12 @@
      inner_puzzle
      solution_or_conditions
      )
+
+  (include *standard-cl-23*)
```

Comment:
there's a reason why I didn't want to have cl-23 because compiled clvm is much smaller

Reply:
this file was using cl-23 all along I believe. it's needed for the assign and destructuring of arguments in restrict-conditions function